scout/server/blueprints/variants/controllers.py

```diff
@@ -485,6 +489,34 @@ def cancer_variants(store, institute_id, case_name, form, page=1):
     return data
 
 
+def variant_count_session(store, institute_id, case_id, var_type, var_category):
```

Comment:
Al alternative to this would be to save these numbers in the database in the case object, so they'll be available all over the app. For instance in the institute page we could display the cases with their number of variants and so on.